src/components/gamewindow/gamewindow.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from "react";
+import axios from "axios";
+import styled from "styled-components";
+
+class GameWindow extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      player: {
+        description: "",
+        name: "",
+        players: [],
+        title: "",
+        uuid: ""
+      },
+      message: ""
+    };
+  }
+
+  componentDidMount() {
+    const headersAuthorization = {
+      headers: { Authorization: `Token ${localStorage.getItem("key")}` }
+    };
+    console.log(localStorage.getItem("key"));
+    axios
+      .get(
+        "https://lambdamud-alee.herokuapp.com/api/adv/init",
+        headersAuthorization
+      )
+      .then(response => {
+        this.setState({ player: response.data });
+        console.log(this.state.player);
+      })
+      .catch(err => console.log(err));
+  }
+
+  handleTextChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleLogout = () => {
+    localStorage.clear();
+    this.props.history.push("/login");
+  };
+
+  submitCommand = event => {
+    event.preventDefault();
+    const headersAuthorization = {
+      headers: { Authorization: `Token ${localStorage.getItem("key")}` }
+    };
+    console.log(localStorage.getItem("key"));
+    const messageArr = this.state.message.toLowerCase().split(" ");
+    console.log(messageArr[0]);
+    if (messageArr[0] === "/s") {
+      messageArr.shift();
+      console.log(messageArr);
+      let mymessage = messageArr.join(" ");
+      console.log(mymessage);
+      axios
+        .post(
+          "https://lambdamud-alee.herokuapp.com/api/adv/say/",
```

Comment:
You should create a variable for your URL and pass that variable to your axios, that way if you need to change anything from your url, you will only need to update your variable and your code will work, instead of hard coding it. :-)